core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -280,5 +283,23 @@ public static boolean isUfsObjectStorage(String ufsAddress) {
         || ufsAddress.startsWith(Constants.HEADER_OSS);
   }
 
+  /**
+   * Gets the value from the static key=value mapping in string format. E.g. mapping
+   * "id1=user1;id2=user2" returns value "user1" with key "id1", returns value null with key
+   * "invalid". Returns null if the given key does not exist in the mapping.
+   *
+   * @param mapping the "key=value" mapping in string format separated by ";"
```

Comment:
I think its fine if we plan to stick to the `;` separator model for other things, I thought it was going to be object store specific. I wouldn't make it generic (work for other symbols aside from `;`) right now because we may end up never need to use any other symbol.